aifc/aifc/__init__.py

```diff
@@ -141,8 +141,8 @@
 __all__ = ["Error", "open"]
 
 
-warnings._deprecated(__name__, remove=(3, 13))
-
+# python-deadlib: Remove deprecation warning
+# warnings._deprecated(__name__, remove=(3, 13))
```

Comment:
How about this?
```suggestion
# python-deadlib: Replace deprecation warning not to raise exception
warnings.warn(f"{__name__} was slated for removal after Python 3.13. Please be aware that you are currently NOT using standard '{__name__}', but instead a separately installed 'standard-{__name__}'.", DeprecationWarning)
```

Reply:
Given python 3.13 is now live, would it be clearer to say "was removed in Python 3.13"?

If I were to read "was slated for removal after Python 3.13" with no additional context, I'd interpret that as meaning it's going to be removed in Python 3.14.

Reply:
```suggestion
# python-deadlib: Replace deprecation warning not to raise exception
warnings.warn(f"{__name__} was removed in Python 3.13. Please be aware that you are currently NOT using standard '{__name__}', but instead a separately installed 'standard-{__name__}'.", DeprecationWarning)
```